test/classes/deitz/inherit-class-record/inherit_mod1.good

```diff
@@ -1,4 +1,6 @@
 inherit_mod1.chpl:17: In function 'foo':
 inherit_mod1.chpl:18: error: unresolved call 'C.x'
-inherit_mod1.chpl:4: note: candidates are: D1.x
-inherit_mod1.chpl:8: note:                 D2.x
+inherit_mod1.chpl:4: note: this candidate did not match: D1.x
+inherit_mod1.chpl:18: note: because method call receiver with type borrowed C
+inherit_mod1.chpl:4: note: is passed to formal 'this: borrowed D1'
+inherit_mod1.chpl:8: note: candidates are: D2.x
```

Comment:
@mppf: While continuing to look at error messages, I find myself thinking that once resolution errors started explaining why the "likeliest" match was not selected, the phrase `candidates are:` in errors like this would be clearer if it said `other candidates are: ` or `additional candidates are: ` to make it clearer that they are distinct from the candidate that was already described.  However, this has the downside of moving the indentation of that list even further to the right.  Any thoughts?

Reply:
I think the general idea sounds good.
If we are worried about the indenting level, we could have it print out "some other candidates are" on a separate line from any other candidates.